client/components/domains/connect-domain-step/connect-domain-step-done.jsx

```diff
@@ -44,7 +42,7 @@ function ConnectDomainStepDone( { className, domain, step, selectedSiteSlug } )
 	let contentLines = [
 		sprintf(
 			/* translators: %s: the domain name that was connected to the user's site (ex.: example.com) */
-			__( "That's it %s has been successfully connected." ),
+			__( "That's it. %s has been successfully connected." ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.